### PR TITLE
edit private notes and emojis

### PR DESCRIPTION
**Pull request by AndrewCLu**

### CODE REVIEW

File: apps/frontend/src/pages/people/[username].tsx [hunk 1]
```diff
@@ -473,27 +524,6 @@ const UserProfilePage: React.FC = () => {
...
-
-          {connection?.comment?.emoji && (
-            <div className="flex flex-col gap-2 py-4 px-4">
-              <span className="text-sm font-semibold text-primary font-sans">
-                Your Label
-              </span>
-
-              <p className="text-2xl mt-2">{connection?.comment?.emoji}</p>
-            </div>
-          )}
```
Comment (stevenelleman) [inline apps/frontend/src/pages/people/[username].tsx:496]: Removed to reduce the chances the info from being leaked accidentally?

File: apps/frontend/src/pages/people/[username].tsx [hunk 2]
```diff
@@ -186,7 +216,8 @@ const UserProfilePage: React.FC = () => {
       }
 
       setConnection(user.connections[connection.user.username]);
-      handleCloseTapModal();
+      setShowCommentModal(false);
+      await storage.deleteSavedTapInfo();
```
Comment (stevenelleman) [inline apps/frontend/src/pages/people/[username].tsx:220]: Why is the tap info being deleted?

**Review by stevenelleman — APPROVED** [review]
Tested locally and works.

File: apps/frontend/src/pages/people/[username].tsx (hunk 2 above)
Comment (AndrewCLu) [reply apps/frontend/src/pages/people/[username].tsx:220]: this is just saved tap info, its the temporary tap thats saved from when you tap so the info is surfaced for registration, etc.

File: apps/frontend/src/pages/people/[username].tsx (hunk 1 above)
Comment (AndrewCLu) [reply apps/frontend/src/pages/people/[username].tsx:496]: yeah exactly

**Pull request merged**